### DISCUSSION TITLE
The result would exceed the internal working limit of 100000 rows | changing the default result row limit

### DISCUSSION THREAD

**Discussion by asitha-w** [datalust/seq-tickets#2120] [Help] [answered]
The question is to know the possibility of changing the default row limit of 100,000?
some of our use-cases require people to export data, sometimes exceeding the 100,000 row limit, so we would like to know if this default can be changed.

thank you

**Comment by liammclennan** [comment]
Hi @asitha99x ,

What are you trying to do, how are you trying to do it and what result are you getting?

**Reply by asitha-w** [reply]
(one of our legacy way of working requires the complete data for a given time window - this sometimes exceeds the 100K limit.)

how:
we select **one signal** and a time window say **one day** (eg Today) : this loads data ( paginating ) to the UI.
But when we try to export the data using export as CSV . we are getting the below as the export ( only option is to narrow-down the time window )

result:
```
"Error"
"The result would exceed the internal working limit of 100000 rows."
```


workaround: 
cli works - but we would like to keep the users on the UI

**Reply by nblumhardt** [reply, marked as answer]
Hi Asitha, thanks for the follow-up!

Unfortunately the CLI will be the only option for this currently, the UI doesn't have any mechanism to paginate file downloads.

Another option might be to download 12 hours at a time, and combine these using a script or text editor.

The internal row limit is an important part of keeping the server stable, so we're hesitant to address the issue by allowing it to be modified - the intention is that clients should partition or paginate requests past this size.

**Reply by asitha-w** [reply]
thank you @liammclennan @nblumhardt for the explanation,
yes we agree. keeping things stable is important.
